### PR TITLE
Added support for Butterfly terminal

### PR DESCRIPTION
**Pull request by ecwpz91**
Updated documentation to have the user deploy a containerized TTY.

### CODE REVIEW

**Comment by dudash** [comment]
So is the plan to run butteryfly in containers on the cluster and server 1 user per container or several?

File: content/workshops/openshift_101_dcmetromap/lab1-welcome.md [hunk 1]
```diff
@@ -73,7 +48,7 @@ You just created a project using the web console, let's tell the terminal comman
 > <i class="fa fa-terminal"></i> Type the following command to use the demo project:
 
 ```bash
-$ oc new-project demo-{{< span userid "YOUR#" >}}
+$ oc project demo-{{< span userid "YOUR#" >}}
```
Comment (dudash) [inline content/workshops/openshift_101_dcmetromap/lab1-welcome.md]: good catch - in workshops I've run students always seem to not be in the right project and having issues. I assumed they had just skipped this step, but it's actually a typo.

**Comment by dudash** [comment]
OK, so this is part of the work to get this running on RHPDS?  A little sad to see the wetty container go away - but otherwise, yeah, this looks good to me.

**Comment by ajacocks** [comment]
Reviewed with Mike Surbey; the web interface is used to deploy this terminal, which is the same one used in cockpit

**Pull request merged**